nexus/db-queries/src/db/datastore/instance.rs

```diff
@@ -140,6 +140,21 @@ impl DataStore {
         .map_err(|e| public_error_from_diesel_pool(e, ErrorHandler::Server))
     }
 
+    /// Fetches the instance with the given UUID
+    pub async fn instance_get(
```

Comment:
Same as with disks -- the lack of authz check here seems like a pretty big footgun (even if it's appropriate for omdb).

Reply:
This should be addressed now.